galaxy_importer/ansible_test/runners/local_image.py

```diff
@@ -15,11 +15,45 @@
 # You should have received a copy of the Apache License
 # along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
 
+import subprocess
 
+from galaxy_importer import exceptions
+from galaxy_importer.ansible_test.builders.local_image_build import Build
 from galaxy_importer.ansible_test.runners.base import BaseTestRunner
 
 
 class LocalImageTestRunner(BaseTestRunner):
     """Run image locally with docker or podman."""
     def run(self):
-        pass
+        build = Build(
+            self.filepath,
+            f'{self.metadata.namespace}-{self.metadata.name}-{self.metadata.version}',
+            self.log)
+
+        build.build_dockerfile()
+        build.copy_collection_file()
+
+        tag = build.build_image_with_artifact()
```

Comment:
minor... ideally, these could be encapsulated into a a single method that returns the tag, so runner wouldn't know about inner workings, something like`.build_image()`, and things like `copy_collection_file()` can be an internal class methods